### PR TITLE
Open vote phase

### PR DESCRIPTION
**Pull request by anativ**
This closes the "CFP" and opens the "Votes"

### CODE REVIEW

File: client/src/components/Avatar.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ const Avatar = ({ id, name, picture, sessions, onLogout }) => (
     <ul className={s.menu}>
       <div className={s.menuInner}>
         {/*{ sessions.length > 0 && <Link className={s.menuItem} to={`/speaker/${id}`}>My Profile</Link> }*/}
-        <div className={s.menuItem}><Link to="/profile">MyProfile</Link></div>
+        <div className={s.menuItem}><Link to="/profile">My Profile</Link></div>
+        <div className={s.menuItem}><Link to="/my-votes">My Votes</Link></div>
```
Comment (rantav) [inline client/src/components/Avatar.js]: Add to mobile menu as well

File: client/src/components/SpeakerPage.css [hunk 2]
```diff
@@ -21,4 +21,8 @@
 
 .socialLinks a:not(:last-child) {
     margin-right: 60px;
+}
+
+.changeAnimation {
```
Comment (rantav) [inline client/src/components/SpeakerPage.css]: remove this?

File: client/src/features.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-const cfp = true;
-const voting = false;
+const cfp = false;
```
Comment (rantav) [inline client/src/features.js]: you need to disable (or remove) the form at `/cfp`. If someone tries to hack by calling the JS function that submits or sends an HTTP request - that's ok, we'll live with that, but the form should be removed.

**Comment by anativ** [comment]
The proposals are sticky per "user" so if guest will refresh he will get a "new order" but the same user will see the same order on mobile / desktop

File: client/src/components/Avatar.js (hunk 1 above)
Comment (anativ) [reply client/src/components/Avatar.js]: done

File: client/src/components/SpeakerPage.css (hunk 2 above)
Comment (anativ) [reply client/src/components/SpeakerPage.css]: changed it to 2s

File: client/src/features.js (hunk 3 above)
Comment (anativ) [reply client/src/features.js]: now there a message "Reversim Summit 2018 - CFP is closed!"

**Comment by rantav** [comment]
@anativ feel free to merge when you're ready

**Pull request merged**